src/audio/drc/drc_math_lib/asin/c_fixpt/main.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * File: main.c
+ *
+ * Author :Shriram Shastry
+ * Copyright (c) 2016, Intel Corporation All rights reserved.
+ */
+
+
+
+/* Include Files */
+#include "main.h"
+#include "drc_asin.h"
+
+/*
+ * Arguments    : int argc
+ *                const char * const argv[]
+ * Return Type  : int
+ */
+int main(int argc, const char * const argv[])
```

Comment:
If you need unit tests cmocka would be good I think. See SOF/test. Or to tools/testbench/? Also separate test stuff for separate commit.